### PR TITLE
186 reorder state variable declaration

### PR DESCRIPTION
**Pull request by lirona**

### CODE REVIEW

File: .solhint.json [hunk 1]
```diff
@@ -3,12 +3,16 @@
...
       "no-simple-event-func-name": "off",
       "two-lines-top-level-separator": "off",
       "mark-callable-contracts": "off",
-      "reason-string": "on",
-      "compiler-version": ["error","^0.8.6"],
-      "func-visibility": ["error",{"ignoreConstructors": true}]
+      "reason-string": ["error", {"maxLength": 64}],
+      "compiler-version": ["error", "^0.8.6"],
+      "func-visibility": ["error", {"ignoreConstructors": true}],
+      "max-line-length": ["warn", 179],
```
Comment (lirona) [inline .solhint.json:13]: made it too long for now, need to change to 79 later when we lint everything

Comment (lirona) [reply .solhint.json:13]: or i can do it now in this pr

Comment (jellegerbrandy) [reply .solhint.json:13]: please do not change the linter rules :)

File: contracts/vaults/Base.sol [hunk 2]
```diff
@@ -282,11 +298,15 @@ contract Base is OwnableUpgradeable, ReentrancyGuardUpgradeable {
...
     }
 
-    function setPoolsLastProcessedTotalAllocPoint(uint256 _pid) internal {
-        uint globalPoolUpdatesLength = rewardController.getGlobalPoolUpdatesLength();
-        if (globalPoolUpdatesLength > 0) {
-            poolInfos[_pid].lastProcessedTotalAllocPoint = globalPoolUpdatesLength - 1;
-        }
+    function getDefaultBountySplit() public pure returns (BountySplit memory) {
+        return BountySplit({
+        hackerVested: 6000,
```
Comment (jellegerbrandy) [inline contracts/vaults/Base.sol:303]: a linter warning

**Pull request merged**